<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSlippembayaran</class>
 <widget class="QWidget" name="FormSlippembayaran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kodePembayaranLabel">
      <property name="text">
       <string>Kode Pembayaran</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodePembayaranLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kodeMitraLabel">
      <property name="text">
       <string>Kode Mitra</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="kodeMitraLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="kodeBarangLabel">
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="kodeBarangLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hargaLabel">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="hargaLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="kuantitiLabel">
      <property name="text">
       <string>Kuantiti</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="kuantitiLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="totalHargaLabel">
      <property name="text">
       <string>Total Harga</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="totalHargaLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableSlipPembayaran">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>821</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
